src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/VSMSBuildNuGetProjectSystem.cs

```diff
@@ -929,5 +929,25 @@ public virtual IEnumerable<string> GetDirectories(string path)
                        select p.Name;
             });
         }
+
+        public bool IsNetStandardMSINeeded()
```

Comment:
As discussed offline, this is not a public public method, and for our understanding, name is self explanatory. I can add a comment down line to specify when we think MSI is installed.